dietpi/dietpi-software

```diff
@@ -12417,7 +12417,7 @@ _EOF_
           Download_Install 'https://ci.nukkitx.com/job/GeyserMC/job/Floodgate/job/development/lastSuccessfulBuild/artifact/bukkit/target/floodgate-bukkit.jar' /opt/papermc/plugins/floodgate-bukkit.jar
       G_EXEC cd /opt/papermc
       G_EXEC_OUTPUT=1 G_EXEC java -jar /opt/papermc/paperclip.jar nogui
-      G_CONFIG_INJECT 'eula=[[:blank:]]' 'eula=true' /opt/papermc/eula.txt
+      G_CONFIG_INJECT 'eula=false' 'eula=true' /opt/papermc/eula.txt
```

Comment:
Use `G_CONFIG_INJECT 'eula=' 'eula=true' /opt/papermc/eula.txt` here. `[[:blank:]]` is a placeholder/wildcard for any kind of space, but this option obviously does not contain a space.

Compare with below:
```
G_CONFIG_INJECT 'auth-type:[[:blank:]]' 'auth-type: floodgate' /opt/papermc/plugins/Geyser-Spigot/config.yml
```
Here, due to YAML syntax, a space is always present/required between the name of the setting and the value, so we can include that into the matching pattern to be sure that only the exact setting we want is matched. In this case  `'auth-type:'` would be enough as well, since a `:` in YAML is a strict separator between settings name and value, AFAIK, same as the space. However e.g. `'auth-type'` could be in theory not enough, as there might be a setting now or in the future that is named 'auth-type-strict' or something like that, maybe to remove an otherwise present setting in the web interface, or that each user can/cannot choose it's own authentication type or such (just a dumb example, but you get the point).

Good practice is to define the matching pattern as specific as possible, at best oriented at the syntax of the config file type, but to not include any part of the value (true, false, floodgate) so that the setting is set, regardless which value was present before.